### DISCUSSION TITLE
在哪里设置allow_llm_to_see_data=true啊？

### DISCUSSION THREAD

**Discussion by thegenway** [vanna-ai/vanna#586] [Q&A]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Error logs/Screenshots**
If applicable, add logs/screenshots to give more information about the issue.

**Desktop (please complete the following information where):**
 - OS: [e.g. Ubuntu]
 - Version: [e.g. 20.04]
 - Python: [3.9]
 - Vanna: [2.8.0]

**Additional context**

**Comment by nverseui** [comment]
VannaFlaskApp __init__(self, vn, cache: Cache = MemoryCache(),
                    auth: AuthInterface = NoAuth(),
                    debug=True,
                    allow_llm_to_see_data=False,
                    logo="https://img.vanna.ai/vanna-flask.svg",
                    title="Welcome to Vanna.AI",
                    subtitle="Your AI-powered copilot for SQL queries.",
                    show_training_data=True,
                    suggested_questions=True,
                    sql=True,
                    table=True,
                    csv_download=True,
                    chart=True,
                    redraw_chart=True,
                    auto_fix_sql=True,
                    ask_results_correct=True,
                    followup_questions=True,
                    summarization=True
                 ):
